source/core/replica-set-high-availability.txt

```diff
@@ -15,29 +19,244 @@ Replica Set High Availability
    :depth: 1
    :class: singlecol
 
-:term:`Replica sets <replica set>` provide high availability using
-automatic :term:`failover`. Failover allows a :term:`secondary` member
-to become :term:`primary` if the current primary becomes unavailable.
+:term:`Replica sets <replica set>` use elections to select the 
+:term:`primary` for the cluster. Replica set elections support
```

Comment:
I would use "primary node" or "primary member", it seems more clear to me, unless that violates other examples.

Reply:
Also, it feels like "Replica sets support high availability by holding elections..." would make more sense.  I guess technically the election is supporting it too, but it feels more like the election and subsequent resumption of normal operations is _how_ replica sets support high availability.